.github/workflows/framework-release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release Framework
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  publish:
+    name: Publish draft
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Download wheel
+        run: |
+          tag_name=$(echo "${GITHUB_REF_NAME}" | cut -c2-)
+          echo "TAG_NAME=$tag_name" >> "$GITHUB_ENV"
+          
+          wheel_name="flwr-${tag_name}-py3-none-any.whl"
+          echo "WHEEL_NAME=$wheel_name" >> "$GITHUB_ENV"
+          
+          tar_name="flwr-${tag_name}.tar.gz"
+          echo "TAR_NAME=$tar_name" >> "$GITHUB_ENV"
+
+          wheel_url="https://artifact.flower.dev/py//${GITHUB_SHA::7}/${wheel_name}"
+          tar_url="https://artifact.flower.dev/py//${GITHUB_SHA::7}/${tar_name}"
+          
+          curl $wheel_url --output $wheel_name
+          curl $tar_url --output $tar_name
+
+      - name: Generate body
+        run: |
+          ./dev/get-latest-changelog.sh > body.md
+          cat body.md
+          
+      - name: Release
+        uses: softprops/action-gh-release@v1
```

Comment:
Should we use the SHA of the action here? Ensuring that when `softprops` reuses the v1 tag, we don't get something we don't want?

Reply:
@tanertopal Done!